src/components/main.tsx

```diff
@@ -0,0 +1,51 @@
+import TopThree from "./topthree";
+import Vote from "./vote";
+import TopTen from "./topten";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+interface TopDogsInterface {
+  breed_name: string;
+  count: number;
+}
+
+export default function Main(): JSX.Element {
+  // topDogs is an array that stores the breed_Names and vote count for the top10 dogs
+  const [topDogs, setTopDogs] = useState<TopDogsInterface[]>([]);
+  // toggleRefresh sets the refresh state to be false to prevent a refresh of the next render
+  const [toggleRefresh, setToggleRefresh] = useState<boolean>(false);
+
+  // GET request to pull the top10 dogs from our database
+  useEffect(() => {
+    async function getTopDogs() {
+      const response = await axios.get(
+        "https://emmalaja-pupvote.herokuapp.com/votes"
```

Comment:
Maybe put the website name in a variable which will make the code more readable and allow you to point to local server when in development (with some extra lines of code added in)